subsys/net/ip/connection.c

```diff
@@ -793,6 +837,15 @@ enum net_verdict net_conn_input(struct net_pkt *pkt,
 
 				continue; /* packet was consumed */
 			}
+		} else if (IS_ENABLED(CONFIG_NET_SOCKETS_INET_RAW) && raw_ip_pkt) {
```

Comment:
Note that few lines above, the code is still checking `IPPROTO_RAW` for packet socket, we need to remove/change those checks from packet socket

Reply:
Yes, do you want me to extend this PR with `AF_PACKET/IPPROTO_RAW` removal?

Reply:
I think we need to remove the `IPPROTO_RAW` from `AF_PACKET`, that was actually the point of the actual issue.